plugins/system76-launch/meson.build

```diff
@@ -19,6 +19,7 @@ shared_module('fu_plugin_system76_launch',
   install : true,
   install_dir: plugin_dir,
   link_with : [
+    fwupd,
```

Comment:
was this linker change actually needed?  I didn't think it would be from the changes you did

Reply:
Yes, it was, or I get an obscure JsonBuilder error

Reply:
I think it's fine, I think the linker and meson version defines how strict the dep is upheld. It's probably safer to have it than not.